app/enterprise/2.1.x/developer-portal/administration/application-registration/azure-oidc-config.md

```diff
@@ -0,0 +1,141 @@
+
+   ![Azure App Registrations](/assets/images/docs/dev-portal/ms-azure-app-reg.png)
+
+1. In **Certificates & secrets**, create a Client secret and save it in a
+   secure location. You can only view the secret once.
+
+1. Under **Manifest**, update `accessTokenAcceptedVersion=2` (default is null).
+   The JSON for your application should look similar to this example:
+
+  ![Azure Manifest](/assets/images/docs/dev-portal/azure-manifest.png)
+
+## Create a Service and a Route in Kong
+
+1. Create a Service example:
+
+   ```bash
+   http put :8001/services/httpbin-service-azure url=https://httpbin.org/anything
+   ```
+1. Create a Route example:
+
+   ```bash
+   http -f put :8001/services/httpbin-service-azure/routes/httpbin-route-azure paths=/httpbin-azure
+   ```
+
+## Map the OIDC and Application Registration Plugins to the Service
+
+Map the OpenID Connect and Application Registration plugins to the **Service**.
+The plugins must be applied to a Service to work properly.
+
+1. Configure the OIDC plugin for the Service:
+
+   ```bash
+   http -f :8001/services/httpbin-service-azure/plugins name=openid-connect config.issuer=https://login.microsoftonline.com/<your_tenant_id>/v2.0 config.display_errors=true config.client_id=<your_client_id> config.client_secret="<your_client_secret>" config.consumer_claim=aud config.scopes=openid config.scopes=<your_client_id>/.default
+   ```
+
+1. Configure the Application Registration plugin for the Service:
+
+   ```bash
+   http -f :8001/services/httpbin-service-azure/plugins name=application-registration config.auto_approve=true config.display_name="For Azure" config.description="Uses consumer claim with various values (sub, aud, etc.) as registration id to support different flows and use cases." config.show_issuer=true
+   ```
+
+1. Get an access token using the Client Credential workflow and convert the token
+   into a JSON Web Token (JWT). You can get the token from Azure or Kong. Replace
+   the placeholder values for `<your_tenant_id>`, `<your_client_id>`, and `<your_client_secret>`.
+
+   - Get a token from Azure:
+
+     ```bash
+     https -f POST "https://login.microsoftonline.com/<your_tenant_id>/oauth2/v2.0/token" scope=<your_client_id>/.default grant_type=client_credentials -a <your_client_id>:<your_client_secret>
+     ```   
+
+   - Get a token from Kong:
+
+     ```bash
+     https -f POST "https://localhost:8443/httpbin-azure/oauth2/v2.0/token" grant_type=client_credentials -a <your_client_id>:<your_client_secret> --verify NO
+     ```
+
+1. Paste the access token obtained from the previous step into
+   [JWT](https://jwt.io), and copy the value for the
+   [aud (audience)](https://tools.ietf.org/html/rfc7519#section-4.1.3) claim to
+   your clipboard. You will use the `aud` value as your **Reference ID** for the next step.
+
+## Create an Application in Kong
+
+1. Log in to your Developer Portal and create a new application:
+   - Select the **My Apps** menu -> **New Application**.
```

Comment:
Same issue as a previous comment; turn this list into a numbered list.

Reply:
Done in commit 772e294